tree/ntuple/v7/test/ntuple.cxx

```diff
@@ -787,3 +787,29 @@ TEST(RNTuple, Descriptor)
    delete[] footerBuffer;
    delete[] headerBuffer;
 }
+
+// Tests ReadV() in RColumn.hxx (the case where a std::string overflows to the next page)
+TEST(RNTuple, ReadString)
+{
+   const std::string_view ntupleName = "rs";
+   constexpr int numEntries = 2500;
+   FileRaii fileGuard("test_ntuple_readstring.root");
+   {
+      auto model = RNTupleModel::Create();
+      auto st = model->MakeField<std::string>("st");
+      auto ntuple = RNTupleWriter::Recreate(std::move(model), ntupleName, fileGuard.GetPath());
+
+      for (int i = 0; i < numEntries; ++i) {
+         *st = "foooooo";
+         ntuple->Fill();
+      }
+   }
+
+   auto ntuple = RNTupleReader::Open(ntupleName, fileGuard.GetPath());
+   auto viewSt = ntuple->GetView<std::string>("st");
+   if (ntuple->GetDescriptor().GetClusterDescriptor(0).GetPageRange(1).fPageInfos.size() < 2) {
+      FAIL(); // This means all entries are inside the same page and numEntries should be increased.
+   }
+   int nElementsPerPage = ntuple->GetDescriptor().GetClusterDescriptor(0).GetPageRange(1).fPageInfos.at(1).fNElements;
+   EXPECT_EQ(viewSt(nElementsPerPage/7), "foooooo");
```

Comment:
I forgot, in gtest, when testing for equality, the arguments go the other way round: `"foooooo", viewSt...`